build/lib/experiments.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+export 'src/experiments.dart' show enabledExperiments, withEnabledExperiments;
```

Comment:
[optional] Would `dart_experiments.dart` or `language_experiments.dart` reduce confusion against something related to build/builder experiments?

Reply:
We could do `language_experiments.dart`, `languageExperiments`, and `withLanguageExperiments`. I don't feel strongly either way, wdyt?

Reply:
After looking again, I'm not sure that "language experiments" is very widely used. We don't use that term at all on https://github.com/dart-lang/sdk/blob/master/docs/process/experimental-flags.md#flag-format-for-cli-based-tools

cc @leafpetersen - do you think we should use the general term "experiments" to refer to Dart language feature experiment flags, or should we use something more specific?

@jakemac53 - I noticed that our [docs are wrong](https://github.com/dart-lang/sdk/issues/41676) when they show how to use the flag. The VM requires the `=`, not a space separated option. Would it be sensible to manually require the same here to reduce confusion on slightly different UX depending on how you run? `package:args` allows either, but we could special case it to check that it's used, or maybe issue a warning?

Reply:
Experiment is fine by me.

Reply:
> Would it be sensible to manually require the same here to reduce confusion on slightly different UX depending on how you run? `package:args` allows either, but we could special case it to check that it's used, or maybe issue a warning?

I think its fine if we are a bit more permissive here than the vm or other tools, it isn't worth sacrificing usability just for the sake of 100% consistency.